test/ProgramScopeConstants/issue_102.cl

```diff
@@ -0,0 +1,29 @@
+// https://github.com/google/clspv/issues/102
+
+// RUN: clspv -cl-std=CLC++ -inline-entry-points %s -o %t.spv
+// RUN: spirv-dis -o %t2.spvasm %t.spv
+// RUN: spirv-val --target-env vulkan1.0 %t.spv
+
```

Comment:
Can you please add some check statements that we generate the constants correctly?